ts/input/tex/base/BaseItems.ts

```diff
@@ -774,6 +774,45 @@ export class NotItem extends BaseItem {
   }
 }
 
+export class NonscriptItem extends BaseItem {
+
+  /**
+   * @override
+   */
+  public get kind() {
+    return 'nonscript';
+  }
+
+  /**
+   * @override
+   */
+  public checkItem(item: StackItem): CheckType {
+    if (item.isKind('mml') && item.Size() === 1) {
+      let mml = item.First;
+      //
+      //  Space macros like \, wrap with an mstyle to set scriptlevel=0 (so size is independent of level)
```

Comment:
typo _wrapped_?

Reply:
Actually, "are wrapped" is probably better.